app/subscriber/src/features/filter-media/utils/alphabetArray.ts

```diff
@@ -0,0 +1,9 @@
+/** used to create an array of the alphabet for the filter media narrow down options */
```

Comment:
I think we should re-think this, or alternatively, de-activate letters for which there are no items underneath that letter when the navigation appears.

Reply:
Agreed; however, that may be an iterative ticket. Just stuck to what I was given for this one!